Formula/arcade-learning-environment.rb

```diff
@@ -17,17 +16,30 @@ class ArcadeLearningEnvironment < Formula
   end
 
   depends_on "cmake" => :build
+  depends_on "gcc" # Python bindings fail to build with clang
```

Comment:
Taking another look, issue is from mixing CLT and XCode.app headers (rather than GCC).
- If possible to force CLT `math.h` rather than XCode `math.h`, issue might be fixed.
- Otherwise, can use XCode for everything instead (`depends_on xcode: :build`).